app/controllers/reports/hic/geographies_controller.rb

```diff
@@ -11,6 +11,10 @@ def show
         merge(GrdaWarehouse::Hud::Project.viewable_by(current_user)).
         merge(GrdaWarehouse::Hud::Project.with_hud_project_type(PROJECT_TYPES)).
         distinct
+
+      date = params[:date]&.to_date
+      @geographies = @geographies.merge(GrdaWarehouse::Hud::Project.active_on(date)) if date.present?
+
       respond_to do |format|
         format.html
         format.csv { send_data GrdaWarehouse::Hud::Geography.to_csv(scope: @geographies), filename: "geography-#{Time.now}.csv" }
```

Comment:
Geography is not part of the HMIS spec any more.  @awisema, can you confirm we don't need to replace it with something else for the 2019 HIC?  I believe it can be removed, and I don't think it works as written.

Reply:
@awisema, since it sounds like we need to support the Geography file, can you take a stab at getting it running?  Should be relatively straightforward.